docs/user/filter-and-process/ottl-transform-and-filter/ottl-filter.md

```diff
@@ -4,14 +4,17 @@ Use filters to drop unwanted telemetry data from a pipeline. Filtering helps you
 
 ## Overview
 
+> [!NOTE]
+> This feature is based on the underlying OpenTelemetry Filter Processor. For more implementation details, syntax, and examples, see the [OTel Filter Processor Documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/README.md). However, bear in mind that some limitations and differences exist, as outlined in the [Limitations](./README.md#limitations) and the [Predefined Contexts](#predefined-contexts) sections.
```

Comment:
I believe we can shorten this:

```suggestion
> This feature is based on the [OpenTelemetry Filter Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/README.md), with some limitations and differences (see [Limitations](./README.md#limitations) and [Predefined Contexts](#predefined-contexts)).
```

And why does this have to be a separate note? Why not include it in the note in line 17?